src/encoded/metadata.py

```diff
@@ -520,8 +523,8 @@ def generate_other_manifest_header(manifest_enum):
 
 
 def generate_file_download_header(download_file_name: str, cli=False):
-    """ Helper function that generates a suitable header for the File download, generating 26 columns"""
-    header1 = ['###', 'Metadata TSV Download', 'Column Count', '26'] + ([''] * 22)  # length 26
+    """ Helper function that generates a suitable header for the File download, generating 27 columns"""
+    header1 = ['###', 'Metadata TSV Download', 'Column Count', '27'] + ([''] * 23)  # length 27
```

Comment:
Use Constants for one or both of the numbers so only need to change in one place?